orchagent/intfsorch.cpp

```diff
@@ -268,6 +268,14 @@ void IntfsOrch::doTask(Consumer &consumer)
                 continue;
             }
 
+            /* Wait for the Interface entry first */
```

Comment:
Do we need to wait in intfmgr as well?

Reply:
intfmgr does two tasks of setting IP address and enslaving interface to VRF, which are independent from one another